src/app.ts

```diff
@@ -325,22 +328,12 @@ export class AlgorandApp extends BaseApp {
     const scopeBuffer = Buffer.from([metadata.scope])
     const encodingBuffer = serializeEncoding(metadata.encoding)
     const dataBuffer = decodedData
-    let domainBuffer = Buffer.from([])
-    if(signingData.domain) {
-      domainBuffer = Buffer.from(signingData.domain)
-    }
-    let requestIdBuffer = Buffer.from([])
-    if (signingData.requestId) {
-      const requestIdHexStr = Buffer.from(signingData.requestId, 'base64').toString('hex');
-      requestIdBuffer = Buffer.from(requestIdHexStr, 'hex');
-    }
-    let authDataBuffer = Buffer.from([])
-    if (signingData.authenticationData) {
-      authDataBuffer = Buffer.from(signingData.authenticationData)
-    }
+    const domainBuffer = signingData.domain ? Buffer.from(signingData.domain) : Buffer.from([])
```

Comment:
Really silly thing, but this could be shortened to 
const domainBuffer = Buffer.from(signingData.domain || [])